subsys/usb/class/cdc_acm.c

```diff
@@ -403,7 +403,7 @@ static void cdc_acm_do_cb(struct cdc_acm_dev_data_t *dev_data,
 	struct usb_cfg_data *cfg = (void *)dev->config->config_info;
 
 	/* Store the new status */
-	if (status != USB_DC_SOF) {
+	if (status != USB_DC_SOF || status != USB_DC_INTERFACE) {
```

Comment:
I did not fix it exactly like this and just assumed your code did the same thing.
change to the following instead:
`if (!(status == USB_DC_SOF || status == USB_DC_INTERFACE)) {`